### DISCUSSION TITLE
Strelka Problem

### DISCUSSION THREAD

**Discussion by AmgdGocha** [Security-Onion-Solutions/securityonion#6744] [Unsupported Versions]
Hi,
I've installed a distributed environment of SecOnoin on ESXI and everything works just fine, however, Strelka doesn't alert on anything! I've tried to transfer mimikatz over HTTP and it didn't trigger!

I have performed the same test (transferring mimikaz over HTTP) on a standalone deployment and it triggered.

any idea what may be wrong here?

**Comment by dougburks** [comment]
Is Strelka generating any logs at all?
https://docs.securityonion.net/en/2.3/strelka.html#logs
